app/src/main/java/com/nextcloud/talk/adapters/items/UserItem.java

```diff
@@ -186,56 +187,27 @@ public void bindViewHolder(FlexibleAdapter adapter, UserItemViewHolder holder, i
-                    holder.videoCallImageView.setVisibility(View.GONE);
-                    break;
-                case IN_CALL_WITH_VIDEO:
-                    holder.voiceOrSimpleCallImageView.setBackground(
-                            ResourcesCompat.getDrawable(resources, R.drawable.shape_call_bubble, null));
-                    holder.videoCallImageView.setBackground(
-                            ResourcesCompat.getDrawable(resources, R.drawable.shape_video_bubble, null));
-                    holder.voiceOrSimpleCallImageView.setContentDescription(
-                            resources.getString(R.string.nc_call_state_in_call, participant.displayName));
-                    holder.videoCallImageView.setContentDescription(
-                            resources.getString(R.string.nc_call_state_with_video, participant.displayName));
-                    holder.voiceOrSimpleCallImageView.setVisibility(View.VISIBLE);
-                    holder.videoCallImageView.setVisibility(View.VISIBLE);
-                    break;
-                case IN_CALL_WITH_AUDIO_AND_VIDEO:
-                    holder.voiceOrSimpleCallImageView.setBackground(
-                            ResourcesCompat.getDrawable(resources, R.drawable.shape_voice_bubble, null));
-                    holder.videoCallImageView.setBackground(
-                            ResourcesCompat.getDrawable(resources, R.drawable.shape_video_bubble, null));
-                    holder.voiceOrSimpleCallImageView.setContentDescription(
-                            resources.getString(R.string.nc_call_state_in_call_with_audio));
-                    holder.videoCallImageView.setContentDescription(
-                            resources.getString(R.string.nc_call_state_with_video));
-                    holder.voiceOrSimpleCallImageView.setVisibility(View.VISIBLE);
-                    holder.videoCallImageView.setVisibility(View.VISIBLE);
-                    break;
-                default:
-                    holder.voiceOrSimpleCallImageView.setVisibility(View.GONE);
-                    holder.videoCallImageView.setVisibility(View.GONE);
-                    break;
+            Long inCallFlag = participant.getInCall();
+            if ((inCallFlag & InCallFlags.WITH_PHONE) > 0) {
+                holder.videoCallIconView.setBackground(
+                        ResourcesCompat.getDrawable(resources, R.drawable.ic_call_grey_600_24dp, null));
+                holder.videoCallIconView.setVisibility(View.VISIBLE);
+                holder.videoCallIconView.setContentDescription(
+                        resources.getString(R.string.nc_call_state_with_phone, participant.displayName));
+            } else if ((inCallFlag & InCallFlags.WITH_VIDEO) > 0) {
+                holder.videoCallIconView.setBackground(
+                        ResourcesCompat.getDrawable(resources, R.drawable.ic_videocam_grey_600_24dp, null));
```

Comment:
before this used things like: `shape_video_bubble`
Is it better to use the ic_ directly vs. the shape_ ?

Reply:
In this case, we shouldn't use the bubble which has likely been used since the icon would have been on the avatar and depending on the avatar quite hard to read/see. Since you put it to the right of the whole item we are fine since it is on the backgound (`#ffffff`or `#121212`) so we are fine 👍